tests/cases/fourslash/unusedImports14FS.ts

```diff
@@ -0,0 +1,12 @@
+/// <reference path='fourslash.ts' />
+
+// @noUnusedLocals: true
+// @Filename: file2.ts
+//// [| import /* 1 */ A /* 2 */, /* 3 */ { x } from './a'; |]

```

Comment:
The other thing is, I can't decide if that ```/* 6 */``` should stay or go. I think either way is fine, so I don't want to necessarily codify the existing behavior in tests.